h/views/api/config.py

```diff
@@ -78,7 +77,11 @@ def add_api_view(
 
         links.register_link(link, versions, config.registry)
 
-    config.add_view(view=view, **settings)
+    if API_VERSION_DEFAULT in versions:
```

Comment:
This fixes a logic bug in which two views could be registered for a single route, both with `accept="application/json"` (which will raise/not be OK), so we need to register only the view that corresponds to the _default_ API version to respond to this accept value.

Reply:
This will have a nice side effect of breaking if we try to register more than one view to handle a single version for a single route. We'd want breakage in that case :)